docs/API.md

```diff
@@ -45,12 +46,12 @@ The return object are multiple JWTs returned from the auth server, from both the
 
 ### Titles
 
-* A list of known client IDs for convenience. Currently exposes `MinecraftNintendoSwitch` and `MinecraftJava`. These should be passed to `Authflow` options constructor.
+* A list of known client IDs for convenience. Currently exposes `MinecraftNintendoSwitch` and `MinecraftJava`. These should be passed to `Authflow` options constructor (make sure to set appropriate `deviceType`).
 
 Example usage :
 ```js
 const { Authflow, Titles } = require('prismarine-auth')
-const flow = new Authflow('', './', { authTitle: TitlesMinecraftJava })
+const flow = new Authflow('', './', { authTitle: TitlesMinecraftJava, deviceType: 'Nintendo' })

```

Comment:
Should this be 

`const flow = new Authflow('', './', { authTitle: Titles.MinecraftJava, deviceType: 'Win32' })`

Reply:
yeah, fixed